MrtCore/winui_desktop_packaged_app_cpp/winui_desktop_packaged_app_cpp/winui_desktop_packaged_app_cpp.vcxproj

```diff
@@ -18,11 +18,11 @@
     <AppxPackage>false</AppxPackage>
     <ApplicationType>Windows Store</ApplicationType>
     <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
-    <TargetPlatformMinVersion>10.0.18362.0</TargetPlatformMinVersion>
+    <TargetPlatformMinVersion>10.0.19041.0</TargetPlatformMinVersion>

```

Comment:
>10.0.19041.0 [](start = 30, length = 12)

This is 10.0.17763.0 in the wapproj. Would it be good to match that? Do we need to have such a high min version?

Reply:
a mistake - reverted to 18362 to match winui_class_lib